tasks/verify-changed-directories.yml

```diff
@@ -6,14 +6,19 @@ metadata:
 spec:
   params:
     - name: operator_name
-    - name: operator_version
-    - name: git_pr_branch
+    - name: bundle_version
+    - name: pr_head_label
+    - name: git_repo_url
+    - name: base_branch
+      default: main
   steps:
     - name: get-cert-project-id
-      image: registry.access.redhat.com/ubi8-minimal
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
       script: |
         #! /usr/bin/env bash
-        # curl https://api.github.com/repos/operator-framework/community-operators/compare/master...$(params.git_pr_branch)
-        # Operator name and version should correspond to the ancestor directory names. A new release directory should be created.
-        # There should be no additional changes.
-        echo "ok"
+        verify_changed_dirs \
+          --operator_name="$(params.operator_name)" \
+          --bundle-version="$(params.bundle_version)" \
```

Comment:
the CLI has `--bundle_version` argument. Please replace `-` with `_`

Reply:
Would it be better to keep `-` and fix the other args instead? Maybe I'm just a big fan of dashes :smile:

Reply:
@amisstea  You are right, I changed the underscores to hyphens for consistency with other scripts parameters. Related PR: https://github.com/redhat-openshift-ecosystem/operator-pipelines-images/pull/8